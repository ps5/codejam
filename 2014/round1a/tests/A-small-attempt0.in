100
1 2
00
01
5 7
1100010 0111111 0110101 1111110 1001110
0101001 1001011 1111100 0001100 0001000
2 10
1011010111 0000011100
0001111111 1011000011
6 7
1101110 0110100 0110101 0100111 1111101 1000110
1000110 1111011 0001111 1001101 1011011 0000001
10 10
0101000011 1010011111 1000110101 0110101000 1101101001 1111010000 0010000010 1011010110 0000111111 0101101100
0010001001 1010010100 0000100011 1101010101 0111000110 0011000000 1110111110 1000101001 1101111010 0101111111
8 5
01010 10100 00100 10000 11010 10001 00001 00000
11010 11111 01111 01010 00001 01011 10001 11011
3 10
1001001000 0111001011 0010010010
0000011011 1001100011 1010000011
10 10
0010100111 0111110110 0001101011 0101001100 1001111110 1010110001 1100010100 0010010000 1100001001 1111001011
0001101100 0001011011 1100000000 1010110101 1001111010 0100111101 1111011111 1111000010 0110000111 0010100000
6 4
0000 0101 0110 1101 1010 1011
0010 0111 1001 1111 1000 0100
4 4
0000 1011 1110 0111
1111 0100 1010 0011
10 10
0110001111 0010110010 1110101010 1111110101 1001110000 1101010111 1100001001 0011110000 0001001110 0000001101
0110001100 1011001011 1010001001 0111010011 0101110000 1001110100 1000110111 0000001001 0100101110 1111110110
8 8
10011001 00111100 00110110 10110011 01001101 10111001 10010011 01000111
00100111 10100010 00001101 00101101 10101000 11011001 11010011 00000111
6 6
000010 101010 001011 010001 011110 010100
011011 010001 001110 010100 000111 101111
3 6
111001 100000 100001
001110 001111 010111
1 10
0111010000
0001001110
1 5
10001
11110
7 7
1001111 1010000 1100011 0001011 0000101 1101010 0111001
1101011 0101111 0011101 1001110 1000111 0100001 1110100
7 7
1011000 1111111 0001101 0011111 0000100 1101010 0011100
1111111 0001010 0010001 0001001 0011000 1001101 1101010
10 10
1111110000 1001101111 1000010110 0100011001 0110100000 1100010011 1101101011 0011001101 0011100100 0010011110
0101001101 0110101110 1110100100 1001110000 1001011001 0011010010 1000100011 0010101011 1100011101 0111010110
2 9
001101001 001101011
100110000 100110010
4 6
111010 100011 100001 111000
111100 111110 100101 100111
8 8
00001111 10011010 11000111 10100010 10110100 10101100 10100001 00111111
11011100 01011110 01100100 01100111 10000110 10000000 00100100 00010110
1 9
101111000
011111100
4 5
11001 11000 11011 01111
01110 11001 11000 11010
6 4
0000 0100 1111 1000 0001 1100
1010 1110 0010 0111 0110 1001
4 8
01011111 01011001 00110110 00110000
11011001 11011111 10110110 10110000
6 10
1110011110 0110011011 1110001011 0010010001 1010010100 1010000001
1101011010 1101001111 1001010000 0001010101 1001000101 0101011111
3 6
001010 000111 100100
101000 001011 100101
5 10
1110110101 0011111100 1010011100 1110111011 0100100111
1001101110 0010001001 1011001010 0001010101 0111001001
1 3
111
100
10 10
0101000011 1010111000 0011011110 0110101001 0100110111 1010101110 1001001000 0101100111 1001010001 1110010100
0100100101 1000010011 1011101100 1000001010 0101110101 0111101011 0010011100 0100000001 1111010110 1011111010
10 10
0100000000 1001011001 1110101110 0010110011 1111000101 0001111101 1001101011 1000100110 0111011010 0110010100
1111001101 1010110000 0100001000 0010010100 1001111110 0101100011 0010100110 0011101011 1101011001 1100010111
10 10
1101110001 0011011100 0011101111 0110100010 0101000111 1110010101 1001111000 1110000010 0000101000 1000011111
0101011000 0011000101 1110011101 1011110010 1000110101 0010100010 0101001111 1101111000 0110101011 1000000110
10 10
1000111101 0100111110 0101000001 0111110101 1011001000 1010001111 1100000110 0011110001 1010110010 0101001010
0010010101 0001011110 1010101101 1101100011 0011100001 0011101010 1100011001 1110010110 1100100100 0101011010
4 5
00001 00010 00011 10101
11100 11101 01011 11111
10 10
1011010100 0110000011 0101000111 1001101111 1000110010 0101011100 0110111000 1110111001 1001001111 0010100000
0011010110 1101100111 0000001001 1110000001 1100011010 0011101101 0111110101 1011101100 1100111010 0000010010
2 3
010 000
111 101
2 8
01011010 10100100
01010011 10101101
9 7
0001100 1110010 0101011 0000111 1111110 1101011 1101101 0001110 0110110
0110101 1000101 0100001 0111101 0100010 0011100 1010110 0001101 0000000
2 6
111000 011110
100001 010000
10 10
0001011000 1111100001 1011101100 0110100101 0011101010 1101010111 0100010000 1010011011 1100001111 0000110110
1001110101 0101001100 0100010100 0110100010 0010101111 0011011000 1111100110 1010101001 1101010011 1000011011
2 6
101000 001111
100011 000100
3 7
1011010 1101111 1001110
0010010 0100111 0110011
10 10
1101111000 1100100011 0001001110 0110000101 1000001110 0011010101 0111111010 1010100101 0011111001 1100010010
1011010000 0000111011 0110001111 1100001101 0010001100 1101010110 1001111011 0111110000 0010100000 1101100111
2 5
11010 11100
10101 10011
10 10
1110111001 0001011010 0110010000 1100100111 0000001010 0111010101 1000101101 1011001010 1111110101 0001100110
1100101000 0010010101 1000100010 1010111100 0011010000 0101100011 1011110000 0100001111 0101011111 1111001111
4 4
0101 1001 0100 1000
0111 1011 0110 1010
10 10
1010111100 0100000001 0101110010 1110000011 1101101001 1000010100 0011111101 1010000010 0001001110 0111111111
0100000010 0010110011 1101111110 1110010100 0110001111 0111111100 1001111111 0000000000 1001000001 1011101001
8 10
1101101001 0111111100 0110011101 0010111011 0011101101 0010110000 0011011110 0001110011
0010111001 1101101011 0011011100 0011101111 0001110001 0110011111 0010110010 0111111110
2 9
000000010 001101101
000110001 001011000
4 6
110111 100111 101010 000010
010111 011010 000111 110010
4 10
0111011110 1000100011 0100000101 0001111011
1111001110 0011101000 0110010110 0000110011
4 9
111000110 111100001 010001010 110010111
111111111 101101000 101110111 111101000
4 9
010010010 011001010 100101011 010100000
001111100 110010111 001000100 101111011
10 10
1110101111 1101101000 1101111010 0001000111 0000001000 0010100111 1110110000 0000010001 1011010110 0111011101
0100010110 1110000111 0010101000 0001101111 1000011101 0010111010 1101100111 0001110000 1111001000 1111010001
10 10
1100111011 1010111100 1011011101 0010000011 0111000101 0101111110 1110010000 0000100000 1101001110 0001100011
1001100000 0101111011 0111010000 1011000011 0100001110 1000100011 1100111110 0010011101 1110000101 0011111100
10 10
1100100100 1010000011 1101100000 1001001011 0101110101 0110010110 0010011101 0011101010 0100011111 1011111000
1111101101 1011100110 0011111000 1100001110 0010000011 0000110000 0101011111 1101100100 0100011011 1010010001
7 8
11111111 01100010 01010001 11011101 01101000 10011011 11110100
10011011 10111101 00100111 11101101 00110000 11110100 10001111
10 10
1101111000 0100111100 1010000011 0111001100 1010010011 0010100011 1000110101 0111110111 1101001010 0001001100
1110110100 1000001111 0011000000 0100001001 0011010000 1110001111 0100111011 0001110110 1101111111 1011100000
10 10
0101001010 0111101111 1000010100 0000100011 1110110000 0001000111 1010101110 0101011100 1010010001 1111111001
1001100101 0110101000 0100011011 0110111110 0011010111 1011100000 1100001101 1101000100 0010110011 1001011010
10 10
0101000000 1101110101 0000101011 0001011100 1010111001 1000110100 1011001011 0110001110 0110110011 1111000110
1000000010 0000110111 0111111011 0101110110 1011110001 0110001001 0010000100 1011001100 1101101001 1100011110
10 10
1011111001 1001110010 0100110010 1101000111 0011011100 1110001110 0000101001 0000100100 1111011101 0110000011
1001011010 1010100010 0111110000 1111111101 0110100011 1001010111 0010000111 0100111001 1101001100 0000001100
10 10
1001111101 1001111111 1001100111 0010000110 1101111001 0010011100 1001110001 0110011000 0010011110 0010010000
1101101101 1101110101 1101100011 0110000010 1101101111 1001101011 0010001010 0110010100 0110001100 0110001110
1 2
01
11
1 10
1010001111
1011011100
10 9
111111001 111111110 101110110 001011000 000000001 110110001 001101000 000111010 011101111 111100001
111011110 011001111 001111000 000100001 111000001 001001000 000011010 111011001 101010110 110010001
4 9
001000010 001111100 001100111 111111111
101110011 100100101 010111101 111001100
5 6
100010 010101 011001 110000 111001
100111 100001 010001 001101 100100
7 5
01110 11111 11001 00001 10101 01111 01001
10110 11001 00010 11000 01000 01110 11110
5 9
010101001 000011100 111111111 010011001 001101101
111000001 110101001 110111101 010111001 100110100
9 7
0011101 1010111 1101001 0010110 1001111 1011111 1110011 1110000 0101111
0011011 1101011 0110111 0110100 0010011 1010010 0101101 0001011 1011001
10 10
1101010011 0100001010 0111100100 1000101101 0010111011 1011111110 1100100100 0101011001 1010010011 0011000100
0000110001 0001001110 1001110100 1000011001 0101101110 1010100111 1111011001 1110101110 0111010011 0110000000
10 7
0101111 0010000 0110110 0000011 0110111 0001111 0011001 1000110 1010101 1000101
1100111 0111011 1010001 1111101 1001001 1101110 1110001 0111000 1001000 0101011
10 10
1011010011 1000001011 0110110111 0101001100 0100110100 0000101010 1101011101 0011101010 1110110000 1011000101
0000011001 1110000110 0011000001 1111111110 1000100000 0000001111 0101111010 1011100000 1101111101 0110010111
7 6
000011 000110 111001 011000 001101 010010 010110
111010 001000 110110 010111 010100 011011 000111
1 10
1111101100
0111011101
5 6
110101 010000 001111 111010 111000
111111 001001 110010 010100 110000
1 8
10001010
01011011
2 8
11001100 00100011
01100110 10001001
8 8
10001011 11000001 01110100 00111110 11001011 01111110 00110110 10000011
00110110 11000001 01110100 11001001 10001001 10000011 00111100 01111100
10 10
0011000011 0101000001 1000011100 0101110111 0110101101 1111000110 1110011010 1011111110 1000101001 0000110000
0010000011 1001011000 0011011111 1100101001 1111011010 0111100111 1010110100 1001101110 0100000101 0100110000
8 8
01001010 00111111 00110101 10010101 11000001 00011110 01101011 01100001
10010111 10111100 10011101 00110111 11001001 11000011 01100011 11101000
4 8
00010010 10101010 11101000 01010000
00010010 10101010 01010000 11101000
10 10
1101011011 0010101100 1011000111 0000110000 0100100101 1011111010 1110010011 1111000100 0001101001 0100011110
0000010010 1100101101 1110110001 0001000101 0101111011 0011011010 1111101000 0101000110 1010100100 1010011111
5 10
1110100111 0010000110 1110001111 0100110011 0000101100
0001001100 0101010011 0111111001 1011110000 1011011000
10 10
0100110001 1011000010 0001100110 1110101101 0110000110 0001000100 1110111000 1100011111 1011111001 0001011011
0110101101 0011101100 1001100101 0001001011 1100010000 1011010010 1100110010 0110010110 0011111001 1100001111
6 6
111101 001101 001111 010100 111111 010110
100110 010110 001111 010100 100100 001101
7 9
111100110 000110111 100010111 010010010 110110111 001111100 111011011
000100110 000011011 110000001 101101111 111001010 011101010 001001010
3 8
01011001 11010111 01000100
00001000 10000110 10011011
1 9
100001010
101100101
2 9
111000001 001101101
000101101 110000001
5 9
111111110 001111001 000111010 011000001 011010000
000011100 100100011 011100111 000001101 010100100
10 10
0011101111 1111100010 1000111001 0110010110 0010111111 1110110010 1110011000 1111001000 1001101001 0111000110
1011000100 0010011010 0101101011 0010110000 1111101101 1010010100 0100111011 1110111101 0011001010 0011100000
4 8
01100100 01001011 10010100 11011101
11100100 00010100 01011101 11001011
10 10
1011001010 0000111101 0101000100 0000000000 1010110111 0111111000 1100001111 1100110101 1011011011 0111100010
1110011011 0101111101 1001000100 1110100110 0010010011 1001011110 1011100010 0010101001 0100010001 0101101100
1 5
11001
00000
10 10
0110001101 0101101001 1111100001 1111111010 0100110110 0000010010 1011101101 1010010111 0000011100 1001000010
0101111110 1111111011 1010001101 1110000001 0000000101 1100101110 1010010110 0101110000 0001011010 0011100001
10 10
0100011000 1011101000 0001100110 1000001111 1111010101 1101111111 0011001010 0010110001 0100110100 1110000011
0101000100 1001110110 0110010011 1100011101 0011000001 0100111111 1000100000 1010001010 0011101101 1111111010
4 4
1100 1010 0111 1111
0100 1100 0111 0001
9 9
000011011 001111111 100001010 100110110 001101111 101001000 110010111 111111010 111110010
011001001 101100001 111101000 001000000 100000111 110011000 100111011 010100111 011101011
