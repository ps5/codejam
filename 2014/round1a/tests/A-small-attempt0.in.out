Case #1: 1
Case #2: NOT POSSIBLE
Case #3: NOT POSSIBLE
Case #4: NOT POSSIBLE
Case #5: 4
Case #6: 3
Case #7: NOT POSSIBLE
Case #8: 5
Case #9: 1
Case #10: 1
Case #11: 6
Case #12: 3
Case #13: 2
Case #14: 4
Case #15: 6
Case #16: 4
Case #17: 2
Case #18: 3
Case #19: 7
Case #20: 5
Case #21: 1
Case #22: NOT POSSIBLE
Case #23: 3
Case #24: 1
Case #25: 2
Case #26: 1
Case #27: 3
Case #28: 5
Case #29: NOT POSSIBLE
Case #30: 2
Case #31: 2
Case #32: 6
Case #33: 6
Case #34: 6
Case #35: 4
Case #36: 5
Case #37: 2
Case #38: 2
Case #39: NOT POSSIBLE
Case #40: NOT POSSIBLE
Case #41: 4
Case #42: 3
Case #43: 6
Case #44: 5
Case #45: 2
Case #46: 3
Case #47: 1
Case #48: 7
Case #49: 1
Case #50: NOT POSSIBLE
Case #51: 2
Case #52: 7
Case #53: NOT POSSIBLE
Case #54: NOT POSSIBLE
Case #55: 4
Case #56: 2
Case #57: 7
Case #58: NOT POSSIBLE
Case #59: 4
Case #60: 5
Case #61: 4
Case #62: 7
Case #63: 3
Case #64: 1
Case #65: 4
Case #66: 1
Case #67: NOT POSSIBLE
Case #68: NOT POSSIBLE
Case #69: 4
Case #70: NOT POSSIBLE
Case #71: 2
Case #72: 3
Case #73: 6
Case #74: 5
Case #75: NOT POSSIBLE
Case #76: 4
Case #77: NOT POSSIBLE
Case #78: 4
Case #79: 3
Case #80: 2
Case #81: 5
Case #82: 3
Case #83: 0
Case #84: 4
Case #85: 8
Case #86: 5
Case #87: 3
Case #88: 8
Case #89: 7
Case #90: 6
Case #91: 1
Case #92: 6
Case #93: 3
Case #94: 1
Case #95: 6
Case #96: 3
Case #97: 5
Case #98: 7
Case #99: 3
Case #100: NOT POSSIBLE
